src/components/multi-action-button/multi-action-button.component.tsx

```diff
@@ -140,12 +152,19 @@ export const MultiActionButton = ({
         ref={buttonContainer}
         onKeyDown={handleKeyDown}
       >
-        {childrenWithProps()}
+        <FormSpacingProvider>{childrenWithProps()}</FormSpacingProvider>
```

Comment:
I definitely understand why we would want to remove any formSpacing here so there are no gaps between the child buttons. However as in the other components where we're doing this, I'd prefer it to be more explicit that we're doing this in order to set `marginBottom` to `undefined` in the children.

Reply:
This will be removed but where it is needed I'll explicitly set the value 👍